internal/common/utils.go

```diff
@@ -14,3 +16,16 @@ var Logger hclog.Logger
 func init() {
 	Logger = base.Logger()
 }
+
+func HandleApiError(err error) bool {
```

Comment:
Method name is confusing. 
I would rename it to 'isGoogle400Error' or something?